### PR TITLE
Added Enabling inbound connections text to topics.

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

To clarify for users about opening up port 8888 on firewall to enable more connections.

## Changes

Added text from Docker topic to Windows, macOS, and Ubuntu topics.

## Checklist

- [X ] My code follows the style of this project.
- [X ] The code compiles without warnings.
- [X ] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [ X] I accept the above linked CLA.

### CODE REVIEW

File: source/mainnet/net/guides/run-node-ubuntu.rst [hunk 1]
```diff
@@ -74,6 +74,23 @@ The ``concordium-mainnet-node`` service that you just installed will be running
...
+If you are running your node behind a firewall, or behind your home
+router, then you will probably only be able to connect to other nodes,
+but other nodes will not be able to initiate connections to your node.
+This is perfectly fine, and your node will fully participate in the
+Concordium network. It will be able to send transactions and,
+:ref:`if so configured<become-a-baker>`, to bake and finalize.
+
+However you can also make your node an even better network participant
+by enabling inbound connections. By default, ``concordium-node`` listens
+on port ``8888`` for inbound connections. Depending on your network and
```
Comment (abizjak) [inline source/mainnet/net/guides/run-node-ubuntu.rst:89]: The ubuntu mainnet node by default listens on 8888, but the testnet node by default listens on 8889. 

I believe the same applies to Windows and Mac nodes. The ports have to be different so both can run at the same time.

**Review by abizjak — CHANGES_REQUESTED** [review]

File: source/mainnet/net/guides/run-node-ubuntu.rst (hunk 1 above)
Comment (dg-concordium) [reply source/mainnet/net/guides/run-node-ubuntu.rst]: I need confirmation on the Windows and Mac. Who should I ask? Or when you say "I believe" is it not just a belief but a fact?

Comment (abizjak) [reply source/mainnet/net/guides/run-node-ubuntu.rst]: I checked again, and it is indeed as I wrote also for WIn & Mac.

**Review by smh1001 — APPROVED** [review]

**Review by abizjak — APPROVED** [review]
Looks good.

I think at some point we should probably revise the first paragraph to make it more likely for people to do this. But it's OK as it is for now.

**Pull request merged**